README.md

```diff
@@ -38,3 +38,7 @@ moduleGraphAssert {
 ```
 ./gradlew generateModulesGrapvizText -Pmodules.graph.print.statistics=true -Pmodules.graph.of.module=:feature-one
 ```
+- Adding parameter `modules.graph.output.gv` saves the graphViz file to specified path (extension `.gv` is added if necessary)
```

Comment:
I think we now should remove the part about the `.gv` extension. After that I will just merge it thanks a lot for the change.